src/utils/pricing.ts

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable prefer-const */
+import { ONE_BD, ZERO_BD, ZERO_BI } from './constants'
+import { Bundle, UniswapPool, Token } from './../../generated/schema'
+import { BigDecimal, BigInt } from '@graphprotocol/graph-ts'
+import { exponentToBigDecimal, safeDiv } from '../utils/index'
+
+const WETH_ADDRESS = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+const USDC_WETH_03_POOL = '0x8ad599c3a0ff1de082011efddc58f1908eb6e6d8'
+
+// token where amounts should contribute to tracked volume and liquidity
```

Comment:
Can you add a link to Github original file in github, so we can keep an eye from time to time if this list changes?